todo-app/src/TodoList.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { inject, observer } from 'mobx-react';
+import { todoStore } from './stores';
+import Todo from './Todo';
+import TodoForm from './TodoForm';
+import loadinImage from './loadingImage.gif';
+
+@inject('todoStore')
+@observer
+
+export default class TodoList extends React.Component {
+
+  state = {
+    editingTodoID: null
```

Comment:
Why is this state not inside your MobX store? Please move it inside your MobX store unless you can think of a good reason why it should not be there.